Source/JavaScriptCore/wasm/WasmBBQJIT.cpp

```diff
@@ -4834,12 +4841,18 @@ class BBQJIT {
             case TypeKind::Arrayref:
             case TypeKind::Sub:
             case TypeKind::RefNull:
-                zero = Value::fromI64(0);
+                emitStoreConst(Value::fromI64(0), m_locals[i]);
                 break;
+            case TypeKind::V128: {
+                Address address = m_locals[i].asAddress();
+                m_jit.store64(Imm64(0), address);
+                address.offset += 8;
```

Comment:
IS the mutation here intentional? Can we just store128?

Reply:
Oh, I guess we totally can store a vector here, I've changed it to a `vectorXor` plus a `storeVector`.